docs/source/java/vector_schema_root.rst

```diff
@@ -107,7 +107,7 @@ Here is an example of creating a VectorSchemaRoot:
 
 Data can be loaded into/unloaded from a VectorSchemaRoot via `VectorLoader`_
 and `VectorUnloader`_.  They handle converting between VectorSchemaRoot and
-`ArrowRecordBatch`_ (a representation of a RecordBatch :ref:`IPC <format-ipc>`
+`ArrowRecordBatch`_ (a representation of a RecordBatch :ref:`IPC <format-ipc>` 
```

Comment:
It seems that we don't need this:

https://arrow.apache.org/docs/java/vector_schema_root.html#vectorschemaroot

> [ArrowRecordBatch](https://arrow.apache.org/docs/java/reference/org/apache/arrow/vector/ipc/message/ArrowRecordBatch.html) (a representation of a RecordBatch [IPC](https://arrow.apache.org/docs/format/Columnar.html#format-ipc) message).

Reply:
So shall I remove this line no 110? 

> It seems that we don't need this:
> 
> https://arrow.apache.org/docs/java/vector_schema_root.html#vectorschemaroot
> 
> > [ArrowRecordBatch](https://arrow.apache.org/docs/java/reference/org/apache/arrow/vector/ipc/message/ArrowRecordBatch.html) (a representation of a RecordBatch [IPC](https://arrow.apache.org/docs/format/Columnar.html#format-ipc) message).

So shall I remove line no 110?

Reply:
> It seems that we don't need this:
> 
> https://arrow.apache.org/docs/java/vector_schema_root.html#vectorschemaroot
> 
> > [ArrowRecordBatch](https://arrow.apache.org/docs/java/reference/org/apache/arrow/vector/ipc/message/ArrowRecordBatch.html) (a representation of a RecordBatch [IPC](https://arrow.apache.org/docs/format/Columnar.html#format-ipc) message).

So shall I remove line no 110?

Reply:
Done @kou